DESCRIPTION

```diff
@@ -75,7 +75,7 @@ Encoding: UTF-8
 LazyData: true
 Config/testthat/edition: 3
 Config/testthat/parallel: false
-Config/Needs/check: rstudio/renv
+Config/Needs/check: rstudio/renv, rstudio/rmarkdown
```

Comment:
@froggleston, to address https://github.com/carpentries/sandpaper/pull/486#issuecomment-1615111701, I am using the devel version of {rmarkdown} in the tests.

Reply:
Disregard this; it turns out the error is coming from inside the house